gdnative-core/src/lib.rs

```diff
@@ -32,78 +32,36 @@
 pub extern crate gdnative_sys as sys;
 #[doc(hidden)]
 pub extern crate libc;
-#[macro_use]
-extern crate bitflags;
 extern crate parking_lot;
```

Comment:
This `extern crate` should be behind the `nativescript` feature (or just removed?). This is causing `cargo build --no-default-features` to fail.

Reply:
Oh yeah I forgot about that one, good catch!